src/main/scala/battleships/Main.scala

```diff
@@ -0,0 +1,96 @@
+package battleships
+
+import scala.io.StdIn.readLine
+import scala.util.Random
+
+object Main {
+  def main(args: Array[String]): Unit = {
```

Comment:
already said this is too long of a main method and should be split into multiples of helper methods